index.js

```diff
@@ -0,0 +1,124 @@
+const recipeObject = {
+  id: 1,
+  title: "Gløgg",
+  picture_url:
+    "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
+  ingredients: [
+    { NAME: "Orange zest", AMOUNT: "0.5" },
+    { NAME: "Water", AMOUNT: "200 ml" },
+    { NAME: "Sugar", AMOUNT: "275 g" },
+    { NAME: "Whole cloves", AMOUNT: "5" },
+    { NAME: "Cinnamon sticks", AMOUNT: "2" },
+    { NAME: "Spice", AMOUNT: undefined },
+    { NAME: "Bottle of red wine", AMOUNT: "1" },
+    { NAME: "Raisins", AMOUNT: "100 g" },
+    { NAME: "Slipped Almonds", AMOUNT: "50 g" },
+  ],
+  description: "Mix everything, heat it, and you are good to go!",
+};
+
+// Recipe Card start
```

Comment:
You would not need this type of start/end comments if you would split your code into functions. For example 
```js
function displayRecipe(recipeObject) {
  // Create title
  // Create image
  // and so on...
}
```

Reply:
corrected , and updated to function , thank you